test/e2e/framework.go

```diff
@@ -120,7 +120,7 @@ const (
 	antreaControllerConfName = "antrea-controller.conf"
 	flowAggregatorConfName   = "flow-aggregator.conf"
 
-	agnhostImage        = "registry.k8s.io/e2e-test-images/agnhost:2.29"
+	agnhostImage        = "registry.k8s.io/e2e-test-images/agnhost:2.40"
```

Comment:
I feel it's better to move this version update to a separate PR if it's not related with Windows change.